.buildkite/linux_jdk_matrix_pipeline.yml

```diff
@@ -1,5 +1,9 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
 
+env:
+  DEFAULT_MATRIX_OS: "ubuntu-2204"
+  DEFAULT_MATRIX_JDK: "adoptiumjdk_17"
```

Comment:
we discussed offline with @alexsapran who pointed out that another alternative for this would be to define it as env vars in the [schedule branch section](https://github.com/elastic/logstash/blob/906c2513c33416e19749121cff4ab1348bf882e2/catalog-info.yaml#L371). In that case, the env vars here will override any values in the schedule.

I think I prefer things as they are here, though, because we won't have different defaults per branch and will not need to define them in catalog-info.yaml.